### PR TITLE
Updates april

### PR DESCRIPTION
**Pull request by jatinbajaj1**
In setup script 

1.)   we are making DIR's    logs,data and logs/database and data/elastic and data/elastic_snapshot
2.)  Entry for vm_map for elasticsearch 
3.)  Bring up  database elasticsearch  and cache  too.

### CODE REVIEW

**Comment by jcrwhite** [comment]
Possibly consider bringing up management after the database's with a sleep before exiting the script to ensure first time migrations have time to run on a low-spec host?

File: README.md [hunk 1]
```diff
@@ -71,7 +71,21 @@ cd /workspace
...
+
+**NOTE:** If you want to do manual setup , then you can skip this step.
+
+Run the script with providing the proper URL of the server.
+
+```
+./setup.sh  http://youmachinehostname/
+
+```
+
```
Comment (jcrwhite) [inline README.md:84]: It seems we're either missing the warning that is present in [this PR](https://github.com/CiscoTestAutomation/xpresso/pull/30) in this readme, or we need to add this check to the setup script.

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        * ) echo "Please answer yes or no.";;
+    esac
+done
+
+
+echo -e "${GREEN}Setup complete!${NC}";
+
+
+docker-compose up -d database cache elasticsearch
+
```
Comment (jcrwhite) [inline setup.sh:57]: possibly start management to run initial migrations for the user with `docker-compose up -d management && sleep 30`  to ensure things don't start to fast on low spec host?

Though maybe we should ask if the user wants the system brought up with a prompt?

File: setup.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    read -p "Do you wish to edit /etc/sysctl.conf?" yn
+    case $yn in
+        [Yy]* ) sed -i '/max_map_count/d' /etc/sysctl.conf; echo 'sysctl -w vm.max_map_count=262144' >> /etc/sysctl.conf; echo "file written"; break;;
+        [Nn]* ) break;;
+        * ) echo "Please answer yes or no.";;
+    esac
+done
+
+
+echo -e "${GREEN}Setup complete!${NC}";
```
Comment (jcrwhite) [inline setup.sh:53]: Printing setup complete and then waiting for containers to come up afterwards could confuse users as the script could appear to hang. maybe change to "Setup complete! \nStarting Xpresso..."

File: setup.sh (hunk 2 above)
Comment (jatinbajaj1) [reply setup.sh:57]: i think we can bring up the management for the users.    its a nice idea..    but let the users     have the power to bring up the containers.

File: README.md (hunk 1 above)
Comment (jatinbajaj1) [reply README.md:84]: Added in script to check  if  "/"   is in the end of string .

File: setup.sh (hunk 3 above)
Comment (jatinbajaj1) [reply setup.sh]: updated

**Review by jcrwhite — APPROVED** [review]

**Pull request merged**